### DISCUSSION TITLE
YouTube API Samples Video Quality

### DISCUSSION THREAD

**Discussion by 1CoolUsername** [amzn/selling-partner-api-samples#63] [Announcements]
Just came across the API; seen a couple videos to get acquainted, I have to ask if it is possible to get this in Higher Quality Output? 360p on a mac M1 looks like blobs.
<img width="536" alt="Screenshot 2024-09-30 at 11 02 20 AM" src="https://github.com/user-attachments/assets/4c080b97-804f-4a15-a500-a2afc245c540">
That's the auto output and I see no other options for HD. It seems that workflow is not uploading anything but 360p; unless its on YouTubes end.
https://www.youtube.com/watch?v=9GjgVYNkLuE

**Comment by 1CoolUsername** [comment]
https://github.com/amzn/selling-partner-api-samples/discussions/36

**Comment by leonardonakagawa** [comment]
Hi @1CoolUsername,

Thank you for engaging with our pricing sample solutions and been following our videos! 

I checked in my laptop as well, and I can upscale the resolution on YouTube and look the code normally in the video. Could you please check if that's a network connection issue on your side? 

Happy to discuss more.

Thanks,
Leonardo Nakagawa
Solutions Architect, Selling Partner API Developer Support

**Reply by 1CoolUsername** [reply]
Thank you; possible it has to do with the relative Chromium build on my hardware; certain decoders are not available on this build by default. There are other videos and channels with resolution options so it must be a relative thing to Chromium.

**Reply by leonardonakagawa** [reply]
Awesome! 

Please do let us know if the sample code explanation is clear, or you have any comment/feedback about it. 

Best,
Leo
